### ISSUE TITLE
Can the pretrained model be used directly on images?

### ISSUE THREAD

**Issue by atharvadeore999** [zhuang-li/FACTUAL#6]
Can we use the pretrained model on images to generate scene graph .If yes please let me know the steps required to perform it.

**Comment by zhuang-li** [comment]
No, the pretrained models are only for parsing texts. To parse images, you need to train the model yourself. There are image annotations in this dataset while I didn't perform related experiments.

**Comment by atharvadeore999** [comment]
Can you please help me by suggesting some good models that will help me generate scene graphs from images? Your help will be  greatly appreciated

**Comment by zhuang-li** [comment]
Sorry, I did some literature survey two years ago. By that time, none of
the image scene graph parsers can work. They just generated a lot of noise.
And this is also why I included the image annotation but I didn’t proceed.
Because I feel the text parser at least can have useful downstream
applications while I couldn’t know what the application would be with a
poorly performed image scene graph parser.

However, that was before the era of LLMs, not sure whether the newest
multimodal LLM can solve the problem. Or whether the image scene graph
parsing problem is still worth solving?

atharvadeore999 ***@***.***>于2024年6月27日 周四上午12:44写道：
